snmp/datadog_checks/snmp/data/profiles/_cisco-generic.yaml

```diff
@@ -178,10 +178,12 @@ metrics:
       - OID: 1.3.6.1.4.1.9.9.147.1.2.1.1.1.3
         name: cfwHardwareStatusValue
     metric_tags:
-      - column:
```

Comment:
Do you think it's safer to also keep the old tag?

Reply:
The cfwHardwareType looks to be index 1
```

cfwHardwareStatusEntry OBJECT-TYPE
        SYNTAX     CfwHardwareStatusEntry
        MAX-ACCESS not-accessible
        STATUS     current
        DESCRIPTION
            "An entry in the table, containing status information
            about a resource."
        INDEX { cfwHardwareType }
    ::= { cfwHardwareStatusTable 1 }
```
http://www.circitor.fr/Mibs/Mib/C/CISCO-FIREWALL-MIB.mib

Reply:
Gotcha, true!